shallow_backup.py

```diff
@@ -296,13 +355,37 @@ def backup_fonts(path):
 	sp.run(copy_ttf, shell=True, stdout=sp.PIPE)
 
 
-def backup_all(dotfiles_path, packages_path, fonts_path):
+def backup_all(dotfiles_path, packages_path, fonts_path, configs_path):
 	"""
 	Complete backup procedure.
 	"""
 	backup_dotfiles(dotfiles_path)
 	backup_packages(packages_path)
 	backup_fonts(fonts_path)
+	backup_configs(configs_path)
+
+
+def reinstall_config_files(configs_path):
+	def home_prefix(path):
+		return os.path.join(os.path.expanduser('~'), path)
+
+	def backup_prefix(path):
+		return os.path.join(configs_path, path)
+
+	configs_dir_mapping = {"Library/Application Support/Sublime Text 2/Packages/User/": "sublime_2",
```

Comment:
let's extract these to a constants file to avoid the repeated dict/list declarations.

Reply:
Will be done in the extra issue right?

Reply:
Since this issue overlaps with the work you're doing, I'll give you the option to run point on that issue as well. If you want to coordinate with @raypatterson77 on getting that done, that's good by me. Otherwise, since it's not too much work, you can just do it and close both issues with this PR.